src/components/ChangeCourtSize/CourtSize.tsx

```diff
@@ -0,0 +1,71 @@
+import { useStyleConfig } from "@chakra-ui/react";
+import { Box, Image, Text, SimpleGrid } from "@chakra-ui/react";
+
+const CourtSize = () => {
+  const styles = useStyleConfig("CourtSize");
+
+  return (
+    <SimpleGrid columns={[2, null, 3]} spacingX="78" spacingY="95" my="78" ml="95" mr="95">
+      <Box sx={styles}>
+        <Box width="100%" height="100%" display="flex" alignItems="center" justifyContent="center">
+          <Image src="./courtSize/court-510m2.jpg" />
+        </Box>
+        <Box transform="translateY(10px)">
```

Comment:
Using margin-top instead of translateY.  The reason is that translateY might cause trouble for later modification of Dom. Normally translate using for animation or transition. Margin using to give white space between dom elements.